core/common/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
In reference to why this is needed: https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#mock-the-unmockable-opt-in-mocking-of-final-classesmethods

Reply:
does this affect other tests? i saw several other tests failed. Could you take a look?

Reply:
Took a look. Latest commits should fix. I make use of the new Mockito#mockStatic methods in 3.4.x and above

Reply:
ok..well.. more issues arose. Latest commit fixes the failures of the last build